src/debian/vmmgrapi/nginx.conf.j2

```diff
@@ -0,0 +1,44 @@
+worker_processes 1;
+user www-data www-data;
+error_log  /var/log/nginx/error.log warn;
+pid /var/run/nginx.pid;
+events {
+  worker_connections 1024;
+}
+http {
+  access_log /var/log/nginx/access.log combined;
+  upstream app_server {
+    server unix:/run/gunicorn.sock fail_timeout=0;
+  }
+  server {
+    listen {{ admin_cluster_ip }}:{{ vmmgr_http_port }} ssl;
```

Comment:
You can use the Mozilla SSL configurator to harden the TLS connection: https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=modern&openssl=1.1.1k&guideline=5.7

Reply:
might be a nice future addition yes